horovod/run/common/util/tiny_shell_exec.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Uber Technologies, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from __future__ import print_function
+
+import six
+import sys
+import traceback
+
+from horovod.run.common.util import safe_shell_exec
+
+
+def execute(command):
+    """
```

Comment:
Is there any reason for not using `safe_shell_exec.py`?

Reply:
I have extracted this code from `_get_mpi_implementation_flags` to make that method testable. Didn't want to refactor the how `_get_mpi_implementation_flags` used `safe_shell_exec.execute(...)` internally.